scripts/check-pricing

```diff
@@ -34,6 +34,7 @@ print_help() {
 DFX_NETWORK="${1:-ic}"
 METHOD_NAME_PATTERN="${2:-.}"
 SIGNER_CANISTER_ID="$(dfx canister id --network "$DFX_NETWORK" signer)"
+LOG_FILE="check-pricing.${DFX_NETWORK}.$(date --iso-8601=second).jsonl"
```

Comment:
Numbers are stored as a sequence of JSON blobs, one per API call, so this uses [the recommended "JSON Lines"  suffix](https://jsonlines.org/).